articles/best-practices/custom-db-connections/anatomy.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Best practices for custom database connection anatomy.
+classes: topic-page
+topics:
+  - best-practices
+  - custom-database
+  - extensibility
+  - database-action-scripts
+  - custom-database-connections
+  - scripts
+contentType: reference
+useCase:
+  - best-practices
+  - custom-database
+  - database-action-scripts
+---
+# Custom Database Connection Anatomy
+
+You typically use a [custom database connection](/connections/database/custom-db) to provide access to your own legacy identity store for authentication (sometimes referred to as *legacy authentication*) or user import through [automatic migration](//users/guides/configure-automatic-migration) (referred to as *trickle* or *lazy* migration).
+
+::: note
+You can also use custom database connections to proxy access to an Auth0 tenant in scenarios where you use Auth0 multi-tenant architecture. For more information, see [Using Auth0 to Secure Your Multi-Tenant Applications](/design/using-auth0-with-multi-tenant-apps). 
+:::
+
+You typically create and configure custom database connections using the [Auth0 Dashboard](/connections/database/custom-db/create-db-connection#step-1-create-and-configure-a-custom-database-connection). You create a database connection and then toggle **Use my own database** to enable editing of the database action scripts. A custom database connection can also be created and configured with the Auth0 [Management API](/api/management/v2#!/Connections/post_connections) and the `auth0` strategy. 
```

Comment:
```suggestion
You typically create and configure custom database connections using the [Auth0 Dashboard](/connections/database/custom-db/create-db-connection#step-1-create-and-configure-a-custom-database-connection). You create a database connection and then toggle `Use my own database` to enable editing of the database action scripts. A custom database connection can also be created and configured with the Auth0 [Management API](/api/management/v2#!/Connections/post_connections) and the `auth0` strategy. 
```

Reply:
The **bold** doesn't really show up too well.

Reply:
Bold for UI elements is our standard.